src/transformers/training_args.py

```diff
@@ -425,6 +429,13 @@ class TrainingArguments:
     report_to: Optional[List[str]] = field(
         default=None, metadata={"help": "The list of integrations to report the results and logs to."}
     )
+    ddp_find_unused_parameters: Optional[bool] = field(
+        default=None,
+        metadata={
+            "help": "When using distributed training, the value of the flag `find_unused_parameters` passed to "
```

Comment:
this one is single backquotes, right? not double?

Reply:
This is for the CLI help, not the sphinx docs :-)

Reply:
Ah! what a bummer with this constant switch between sphinx and .md/text :( thank you!